changelog/1.19.0.md

```diff
@@ -0,0 +1,71 @@
+  uses the following fields: _Project Repository URL_ and _Project Git Service_.
+  Buttons created from prior versions will still work, but a project will not be
+  cloned as part of the flow. In order to automate project cloning, use
+  <https://coder.com/docs/workspaces/workspaces-as-code/templates#workspaceconfigurestart>.
+
+### Features ✨
+
+- web: Introduced resource quotas to organizations. Resource quotas define the
+  maximum resource utilization allowable for each member within an organization
+  for which such quotas are enabled. Modify them at **Manage** >
+  **Organizations** > **Edit Organization**.
+- web: Added a reactivation step for returning users that had been marked as
+  dormant. The reactivation step require the user to agree to consume a licensed
+  seat.
+- web: Added administrative control over dev URL access levels
+- infra: Added client certificate authentication support
+- web: The CVM option when creating a custom environment is now selected by
+  default when its enabled at the site level.
+- web: Improved display and support for errors returned by an IdP during sign-in
+- infra: Changed permissions of `/home/coder/` from _755_ to _744_
+- infra: Workspace builds are now validated when setting policy templates
+- code-server: Includes latest version 3.10.0 of code-server
+
+### Bug fixes 🐛
+
+- web: Fixed issue where the **Edit Workspace** dialog showed the resource
+  selectors as disabled while still allowing modifications
+- web: Fixed issue where the **Edit Workspace** dialog displayed resource
+  allocation incorrectly if the workspace was initially built using a template
+- web: Updated validation logic so that workspaces with invalid names can no
+  longer be created web: Fixed an issue causing web-based Terminals to have a
+  double scrollbar and content cut-off.
+- infra: Fixed issue causing workspace autostart to stop or fail intermittently
+- web: Fixed issue in the **Audit Logs** preventing users from clearing the
+  filters
+- web: A _.csv_ extension is now added to exported audit logs
+
+### Security updates 🔐
+
+- web: Upgraded version of Next.js to 10.2.0.
+- web: Upgraded and addressed CVE-2021-21306 in marked.
+- web: Upgraded and addressed Fix CVE-2021-23368 in postcss.
+- web: client secret is now omitted from GET requests to OAuth configurations.
+- infra: Improved use of prepared SQL statements and added tests to ensure the
+  application is guarded from SQL injection.
+- web: When authenticating with the CLI, a new API key is always generated.
+- web: Fixed authentication for the _watch-update_ workspace endpoint. This was
+  previously open to any authenticated user
+- web: Fixed issue where audit logs for **workspace auto-off** was always
+  attributed to the site admin; the audit log has been corrected to display the
+  user owning the workspace.
+- infra: Added client certificate authentication support. Client certificates
```

Comment:
would like to avoid mentioning this for 1.19, we'll mention it in the next release and I'll write some docs for it

Reply:
Done.